syntax = "proto3";
package message;

message GetVerifyReq {
  string email = 1;
}
message GetVerifyResp {
  int32 error = 1;
  string email = 2;
  string code = 3;
}


service VerifyService {
  rpc GetVerifyCode (GetVerifyReq) returns (GetVerifyResp) {}
}